### DISCUSSION TITLE
The scrollbar of tabcontrol will affect the entire TabItem area, not just the TabItem. Header

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#11955] [Q&A] [answered]
`<ScrollViewer HorizontalScrollBarVisibility="Auto">
					<TabControl TabStripPlacement="Bottom">
						<TabItem>
							<TabItem.Header>
								<TextBlock>TabItem1</TextBlock>
							</TabItem.Header>
							<local:MediaControl></local:MediaControl>
						</TabItem>
						<TabItem>
							<TabItem.Header>
								<TextBlock>TabItem2</TextBlock>
							</TabItem.Header>
						</TabItem>
					</TabControl>
					</ScrollViewer>`
Similar to #6932 
What should I do? I just want TabItem. Header to scroll, not the entire area of TabControl to scroll with it.

**Comment by timunie** [comment, marked as answer]
See https://github.com/AvaloniaUI/Avalonia/discussions/11896

**Reply by pyking-wang** [reply]
Solved it, thank you, Tim.😊
